manifest_reprocessing.py

```diff
@@ -0,0 +1,81 @@
+                           'dir/filename spec and contain hdf/imm files.',
+                'Type': 'Action',
+                'ActionUrl': 'https://develop.concierge.nick.globuscs.info/api/automate/transfer',
+                'ActionScope': 'https://auth.globus.org/scopes/524361f2-e4a9-4bd0-a3a6-03e365cac8a9/concierge',
+                'Parameters': {
+                    'manifest_id.$': '$.input.manifest_id',
+                    'destination.$': '$.input.manifest_destination',
+                },
+                'ResultPath': '$.ManifestTransfer',
+                'WaitTime': 300,
+                'Next': 'CreateFuncXTasks',
+            },
+            'CreateFuncXTasks': {
+                'Comment': 'Fetch a manifest, extract a list of files to process, and create a funcx '
+                           'task list',
+                'Type': 'Action',
+                'ActionUrl': 'https://api.funcx.org/automate',
+                'ActionScope': 'https://auth.globus.org/scopes/facd7ccc-c5f4-42aa-916b-a0e270e2c2a9/automate2',
+                'Parameters': {
+                    'tasks': [{
+                        'endpoint.$': '$.input.funcx_endpoint_non_compute',
+                        'func.$': f'$.input.manifest_to_funcx_tasks_funcx_id',
+                        'payload.$': '$.input'
+                    }]
+                },
+                'ResultPath': '$.CreateFuncXTasksOutput',
+                'WaitTime': 300,
+                'Next': 'RunFuncXTasks',
+            },
+            'RunFuncXTasks': {
+                'Comment': 'Run a funcx task on each file or directory',
+                'Type': 'Action',
+                'ActionUrl': 'https://api.funcx.org/automate',
+                'ActionScope': 'https://auth.globus.org/scopes/facd7ccc-c5f4-42aa-916b-a0e270e2c2a9/automate2',
+                # Fetch the input generated by the previous task
+                'InputPath': '$.CreateFuncXTasksOutput.details.result',
+                'ResultPath': '$.ManifestFuncXTasksOutput',
+                'WaitTime': 300,
+                'End': True,
+            },
+        }
+    }
+
+    required_input = [
+        'funcx_endpoint_non_compute',
+        'manifest_to_funcx_tasks_manifest_id',
+
+        'manifest_to_funcx_tasks_funcx_endpoint_compute',
+        'manifest_to_funcx_tasks_callback_funcx_id'
+    ]
+
+    flow_input = {
+        # Contains tutorial files on /share/godata/ for Globus Tutorial Endpoint 1
+        'manifest_id': '80cae0bb-fe9c-4f91-ac03-93e1ac550b7e',
+        # By default, this will transfer an hdf and imm pair
+        'manifest_destination': 'globus://08925f04-569f-11e7-bef8-22000b9a448b/projects/APSDataAnalysis/Automate/',
+        # Contains tutorial files on /share/godata/ for Globus Tutorial Endpoint 1
+        'manifest_to_funcx_tasks_manifest_id': '80cae0bb-fe9c-4f91-ac03-93e1ac550b7e',
+        'manifest_to_funcx_tasks_funcx_endpoint_compute': '4b116d3c-1703-4f8f-9f6f-39921e5864df',
```

Comment:
The `manifest_id` above should be OK to reuse, but not the funcx endpoint or destination.